src/fmv.cpp

```diff
@@ -25,18 +25,53 @@ static f32 Percent(f32 max, f32 percent)
     return (max / 100.0f) * percent;
 }
 
-static void RenderSubtitles(Renderer& rend, const char* msg, float x, float y, float w, float h)
+static void RenderSubtitles(Renderer& rend, std::vector<const SubTitle*> msg, float x, float y, float w, float h)
 {
+
+	const char* prev_sub = "BUG";
```

Comment:
```if (msg.size() > 1) {``` if this is false then it will appear like that, "" will cause nothing to appear which is better than "BUG", if it should always be set then add an assert or LOG_ERROR if it isn't the case.